.github/workflows/security_scan.yaml

```
@@ -0,0 +1,13 @@
+on: push
+jobs:
+  security:
+    environment:
+      name: test-confluencer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/ruby@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)